core/app/models/spree/order_updater.rb

```diff
@@ -59,7 +59,7 @@ def update_shipments
     end
 
     def update_payment_total
-      order.payment_total = payments.completed.sum(:amount)
+      order.payment_total = payments.joins(:capture_events).sum('spree_payment_capture_events.amount')
```

Comment:
Line is too long. [102/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.